### ISSUE TITLE
pycdio fails to build with due to missing swig?

### ISSUE THREAD

**Issue by jansol** [NixOS/nixpkgs#58809]
## Issue description

Not sure what is going on here as pycdio does list stdenv.swig in buildInputs...
```
swigging swig/pycdio.i to swig/pycdio_wrap.c
swig -python -outdir /build/pycdio-2.0.0 -o swig/pycdio_wrap.c swig/pycdio.i
unable to execute 'swig': No such file or directory
error: command 'swig' failed with exit status 1
builder for '/nix/store/vwlqs9b0whrh8v9zm5ryrqwrm9zid01x-python2.7-pycdio-2.0.0.drv' failed with exit code 1
```

### Steps to reproduce

`nix-env -iA nixos.pycdio`

## Technical details

```
 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.30, NixOS, 19.09pre173349.07b42ccf2de (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2`
 - channels(root): `"nixos-19.09pre173349.07b42ccf2de"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by dotlambda** [comment]
It needs to be in `nativeBuildInputs` because of https://github.com/NixOS/nixpkgs/pull/54182.

**Comment by jansol** [comment]
Closed in c48ad78
